samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala

```diff
@@ -21,23 +21,20 @@ package org.apache.samza.checkpoint.kafka
 
 import java.nio.ByteBuffer
 import java.util
-import java.util.Properties
-
-import kafka.api._
-import kafka.common.{ErrorMapping, InvalidMessageSizeException, TopicAndPartition, UnknownTopicOrPartitionException}
-import kafka.consumer.SimpleConsumer
-import kafka.message.InvalidMessageException
-import kafka.utils.ZkUtils
+import java.util.{Collections, Properties}
 
+import _root_.kafka.consumer.SimpleConsumer
+import _root_.kafka.utils.ZkUtils
 import org.apache.kafka.common.utils.Utils
-import org.apache.kafka.clients.producer.{Producer, ProducerRecord}
-import org.apache.samza.SamzaException
 import org.apache.samza.checkpoint.{Checkpoint, CheckpointManager}
 import org.apache.samza.container.TaskName
 import org.apache.samza.serializers.CheckpointSerde
+import org.apache.samza.system._
```

Comment:
No blanket imports

Reply:
but we do need lots of classes from there: 
SystemConsumer, SystemAdmin, SystemProducer, SystemStream, OutgoingMessageEnvelope and much more..